### PR TITLE
[tree widget] add mechanism for blocklisting svg

### PR DESCRIPTION
**Pull request by chejennifer**
added mechanism to blocklist svgs for prod

### CODE REVIEW

File: server/routes/api/browser.py [hunk 1]
```diff
@@ -142,6 +145,13 @@ def get_statvar_group():
     stat_var_group = request.args.get("stat_var_group")
     places = request.args.getlist("places")
     result = dc.get_statvar_group(stat_var_group, places)
+    if os.environ.get('FLASK_ENV') == 'production':
```
Comment (shifucun) [inline server/routes/api/browser.py:148]: can you put this in configmodule.py, so we have a central place to record the config.

Comment (chejennifer) [reply server/routes/api/browser.py:148]: what config are we recording? the environment?

Comment (chejennifer) [reply server/routes/api/browser.py]: done

File: server/configmodule.py [hunk 2]
```diff
@@ -24,11 +24,16 @@ class Config:
     # Additional stat vars that need to be fetched for place page data.
     # This is only needed for local development when cache is not up to date.
     NEW_STAT_VARS = []
+    BLACKLISTED_STAT_VAR_GROUPS = {}
```
Comment (pradh) [inline server/configmodule.py:27]: BLOCKLISTED

Comment (chejennifer) [reply server/configmodule.py]: done

**Review by shifucun — APPROVED** [review]

File: server/routes/api/browser.py [hunk 3]
```diff
@@ -28,6 +29,11 @@
 NO_MMETHOD_KEY = 'no_mmethod'
 NO_OBSPERIOD_KEY = 'no_obsPeriod'
 
+BLOCKLISTED_STAT_VAR_GROUPS = {
+    "dc/g/Person_EmploymentStatus", "dc/g/Establishment", "dc/g/Person_Industry"
+}
+EMPLOYMENT_NUM_DESCENDENTS = 1759
```
Comment (shifucun) [inline server/routes/api/browser.py:35]: please add a comment with TODO to remove it..

**Review by shifucun — APPROVED** [review]

**Pull request merged**